src/explore-education-statistics-common/src/modules/release/components/ReleaseDataAndFilesAccordionSection.tsx

```diff
@@ -0,0 +1,186 @@
+import AccordionSection, {
+  ToggleHandler,
+} from '@common/components/AccordionSection';
+import Details from '@common/components/Details';
+import SectionBreak from '@common/components/SectionBreak';
+import { Release } from '@common/services/publicationService';
+import { FileInfo } from '@common/services/types/file';
+import classNames from 'classnames';
+import orderBy from 'lodash/orderBy';
+import React, { ReactNode } from 'react';
+import styles from './ReleaseDataAndFilesAccordionSection.module.scss';
+
+interface Props {
+  id?: string;
+  open?: boolean;
+  heading?: string;
+  onToggle?: ToggleHandler;
+  release: Release;
+  renderAllFilesButton?: ReactNode;
+  renderCreateTablesButton: ReactNode;
+  renderDataCatalogueLink: ReactNode;
+  renderDataGuidanceLink: ReactNode;
+  renderDownloadLink: (file: FileInfo) => ReactNode;
+  showDownloadFilesList?: boolean;
+}
+
+const ReleaseDataAndFilesAccordionSection = ({
```

Comment:
As mentioned in Slack, would prefer if we went with the following approach:

- Rename the `ReleaseDataAndFilesAccordion` to `ReleaseDataFilesAccordion`
- Add the 'Related dashboards' section to the bottom of this accordion.

This way we don't have to rearchitect the entire accordion across the admin and frontend and can keep code changes down to a minimum.